### ISSUE TITLE
TBC Raid Bugs - 2

### ISSUE THREAD

**Issue by way2adept** [dalaranwow/dalaran-wow#5553]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: In Serpentshrine Cavern, the raid people tried to fish the hidden boss **The Lurker Below**. After several tries, I fished him (got **The Lurker Above** achieve too 🥇) but for some reason, the boss only flashed above water a second. Similar to A'Q Temple's Ouro, he didn't come out of water surface ever after. I did SSC only once so not sure if the behaviour is eternal.

**Expected behaviour**: Upon successful fishing, The Lurker Below should come out of water where we should be able to fight him.

**Steps to reproduce the problem**:

1. Go SSC
2. Fish the Hidden Boss **The Lurker Below** in an attempt to engage him.
3. He might get bugged and never appear on surface again.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
